docs/en/observability/whats-new.asciidoc

```diff
@@ -3,21 +3,272 @@
+such as Logstash or Kafka, before being ingested into {es}. Likewise, {agent}s can now connect to {fleet-server}.
+They no longer require direct access to {es} or {kib}, allowing for tighter controls in the network layer.
+
+Learn more in our {fleet-guide}/fleet-server.html[{fleet-server} documentation].
+
+[discrete]
+== APM & {fleet} on {ecloud}
+
+We've expanded the capabilities of APM on {ecloud} by adding {fleet}.
+{fleet} provides the ability to manage many thousands of Elastic Agents centrally.
+It makes it easier to get started with the Elastic Stack because you can add integrations to Elastic Agents with just a few clicks,
+allowing you to ingest data from popular services and protect hosts from security threats.
+The {fleet} capabilities are currently a beta release.
+
+In new deployments, APM & {fleet} is turned on by default so that you can get started quickly.
+You also get an instance for free with 512MB of memory.
+This is sufficient for hundreds of events per second in APM and enrolling dozens of Elastic Agents into {fleet}.
+
+[role="screenshot"]
+image::images/7.13-new-apm-fleet.png[APM & {fleet} on {ecloud}]
+
+When you need more than the default 512MB of memory due to increasing usage, it's easy to add more capacity.
+In the Cloud Console, just add more memory by adjusting the instance size on the edit deployment page.
+
+If you have an existing deployment without APM enabled, we encourage you to enable it.
+This will make it easier to try the APM & {fleet} apps. Also, it's free, so there is little reason not to.
+Just add capacity on the edit deployment page.
+
+If you are a heavy user of your APM instance on Elastic Cloud, you should know that {fleet} requires additional memory.
+Before upgrading to 7.13, we encourage you to check your memory usage in the Stack Monitoring app.
+If you have less than 200MB left, we recommend increasing the size of your instance before upgrading to avoid a degradation in performance.
+Then, evaluate the best size after completing the upgrade process.
+
+Learn more in our {ece-ref}/ece-manage-apm.html[APM & {fleet} documentation].
+
+[discrete]
+== Collect logs from AWS Fargate
+
+In this release, we're expanding the AWS Fargate integration with the logs collection.
+Now you can use the {filebeat-ref}/filebeat-module-awsfargate.html[AWS Fargate module] in {filebeat} to
+collect container logs from Amazon ECS on Fargate.
+The module uses the {filebeat} {filebeat-ref}/filebeat-input-aws-cloudwatch.html[`awscloudwatch`]
+input to get log files from one or more log streams in AWS CloudWatch.
+Logs from all containers in Fargate launch type tasks that can be sent to CloudWatch by adding the `awslogs`
+log driver under `logConfiguration` section in the task definition.
+
+[discrete]
+== APM Agents: Improved auto-instrumentation and better support of cloud native services
+
+In 7.13, we've added support for more frameworks when it comes to APM agent auto-instrumentation
```

Comment:
Is it necessary to repeat frameworks? I think this sentence is a little wordy too.

Reply:
Good call. Removed.